extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/config/build/MetricsBuildConfig.java

```diff
@@ -15,7 +15,7 @@ public interface MetricsBuildConfig {
      * This property is not available in the Open Telemetry SDK. It's Quarkus specific.
      * <p>
      * Support for metrics will be enabled if OpenTelemetry support is enabled
-     * and either this value is true, or this value is unset.
+     * and either this value is true.
```

Comment:
```suggestion
     * and this value is true.
```

Reply:
It is either. OpenTelemetry must be enabled, which is by default, and OTel metrics. If either is false, there will be no metrics out.

Reply:
<img width="663" alt="Screenshot 2025-04-30 at 11 03 00" src="https://github.com/user-attachments/assets/5a4c7566-a205-455e-8248-c95cea3d2588" />